spec/models/scheduled_message_spec.rb

```diff
@@ -12,4 +12,15 @@
      it { should validate_presence_of(:time_of_day) }
      it { should validate_presence_of(:title) }
   end
+
+  describe '.date_time_ordering' do
+    it 'should display the messages in chronological order' do
+      m1 = create(:scheduled_message, days_after_start: 1, time_of_day: '2000-01-01 11:00:00 UTC')
+      m2 = create(:scheduled_message, time_of_day: '2000-01-01 16:00:00 UTC')
+      m3 = create(:scheduled_message, time_of_day: '2000-01-01 14:00:00 UTC')
+      m4 = create(:scheduled_message, time_of_day: '2000-01-01 11:00:00 UTC')
+      m5 = create(:scheduled_message, days_after_start: 4, time_of_day: '2000-01-01 11:00:00 UTC')
+      expect(ScheduledMessage.date_time_ordering).to match_array([m1, m2, m3, m4, m5])
```

Comment:
I am going to merge this as-is but wanted to leave a comment for future consideration:

in specs it is helpful to create messages in *not* the order you want them returned, to make sure that the ordering is really working and not just in the default order (which is `created_at`).